<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>548</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="show_add_recipe_dlg_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #E2F778;
border-radius: 10px;
}
QPushButton:hover{
background-color: #E9F79E
}</string>
    </property>
    <property name="text">
     <string>Добавить рецепт</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categories_comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="recipes_listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>256</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="recipe_textEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="show_recipes_Btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #E2F778;
border-radius: 10px;
}
QPushButton:hover{
background-color: #E9F79E
}</string>
    </property>
    <property name="text">
     <string>Вывести рецепты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_cur_recipe_Btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #E2F778;
border-radius: 10px;
}
QPushButton:hover{
background-color: #E9F79E
}</string>
    </property>
    <property name="text">
     <string>Показать выбранный рецепт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_Btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #E2F778;
border-radius: 10px;
}
QPushButton:hover{
background-color: #E9F79E
}</string>
    </property>
    <property name="text">
     <string>Удалить рецепт</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Вы авторизованы как:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_enter"/>
    <addaction name="action_backup"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_question">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_question"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_enter">
   <property name="text">
    <string>Войти в другой аккаунт</string>
   </property>
  </action>
  <action name="action_backup">
   <property name="text">
    <string>Создать бэкап</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
